### PR TITLE
more zxcvbn properties

### PR DESCRIPTION
**Pull request by everwanna**
Hi @Loupehope 

Thanks for your excellent work! I got more properties from zxcvbn because I need them in my project. 

Maybe someone else need them too. Thank you again!

### CODE REVIEW

File: Sources/PasswordChecker/PasswordChecker.swift [hunk 1]
```diff
@@ -7,56 +7,80 @@ public enum PasswordCheckerError: Error {
 }
 
 // For more, available properties - https://github.com/dropbox/zxcvbn#usage
+
 public struct PasswordInfo {
+    public init(guessesLog10: Double, crackTimesDisplay: String, score: Int32, calcTime: Int32) {
```
Comment (Loupehope) [inline Sources/PasswordChecker/PasswordChecker.swift:12]: It is preferable to place init after properties

File: Sources/PasswordChecker/PasswordChecker.swift [hunk 2]
```diff
@@ -7,56 +7,80 @@ public enum PasswordCheckerError: Error {
...
         return .success(passwordInfo)
     }
 }
 
 private extension PasswordChecker {
-    
+
     enum JSKey: String, RawRepresentable {
-        case score
+        case guesses_log10, crack_times_display, score, calc_time
```
Comment (Loupehope) [inline Sources/PasswordChecker/PasswordChecker.swift:81]: Preferably use camelCase:
```swift 
case guessesLog10 = "guesses_log10"
case crackTimesDisplay = "crack_times_display"
```

File: Sources/PasswordChecker/PasswordChecker.swift [hunk 3]
```diff
@@ -7,56 +7,80 @@ public enum PasswordCheckerError: Error {
...
+        self.score = score
+        self.calcTime = calcTime
+    }
+
+    public let guessesLog10: Double
+    public let crackTimesDisplay: String
     public let score: Int32
+    public let calcTime: Int32
+
+    public static let zero = PasswordInfo(guessesLog10: 0, crackTimesDisplay: "", score: 0, calcTime: 0)
```
Comment (Loupehope) [inline Sources/PasswordChecker/PasswordChecker.swift:24]: Wouldn't the name `empty` be more appropriate?

File: Sources/PasswordChecker/PasswordChecker.swift [hunk 4]
```diff
@@ -7,56 +7,80 @@ public enum PasswordCheckerError: Error {
...
-                                 userInputs: [String] = []) -> Result<PasswordInfo, PasswordCheckerError> {
-        
+
+    public func getPasswordScore(_ password: String, userInputs: [String] = []) -> Result<PasswordInfo, PasswordCheckerError> {
+
         let result = jsContext.objectForKeyedSubscript(JSScript.scriptName)?
             .call(withArguments: [password, userInputs])
-        
+
+        guard let guesses_log10 = result?.objectForKeyedSubscript(JSKey.guesses_log10.rawValue)?.toDouble() else {
```
Comment (Loupehope) [inline Sources/PasswordChecker/PasswordChecker.swift:56]: Preferably use camelCase

File: Sources/PasswordChecker/PasswordChecker.swift [hunk 5]
```diff
@@ -7,56 +7,80 @@ public enum PasswordCheckerError: Error {
...
+            return .failure(PasswordCheckerError.unableToParseResult)
+        }
+
+        guard let crack_times_display = result?.objectForKeyedSubscript(JSKey.crack_times_display.rawValue)?.toString() else {
```
Comment (Loupehope) [inline Sources/PasswordChecker/PasswordChecker.swift:60]: I'm not sure if it should be a string here - I get the following variable value - "[object Object Object]". Based on the documentation crack_times_display is a dictionary, so is crack_times_seconds.  Please try to bring this `toDictionary()`.

**Review by Loupehope — CHANGES_REQUESTED** [review]
Hi! Thank you so much for the pull request. But I have a couple of comments, please check them.

**Comment by everwanna** [comment]
Thanks for your comments. Let me try.

File: Sources/PasswordChecker/PasswordChecker.swift (hunk 2 above)
Comment (everwanna) [reply Sources/PasswordChecker/PasswordChecker.swift:81]: I usually keep it simple and consistent. It is OK if you prefer.

**Review by Loupehope — APPROVED** [review]
Thanks so much for the wait and corrections. Good job!

**Pull request merged**